README.md

```diff
@@ -20,3 +22,103 @@ When it gets included in your project it should automatically build the browser
+
+```
+load(source: string | NamedNode | (string | NamedNode)[]): Promise<void>
+```
+
+#### loadDataWithAddAndDelGraph
+
+Parses data from the specified `content`, `additions` and `removals` strings (containing RDF) into this store. The data is structured in `format` mimeType (text/turtle by default) and is parsed into `graph`.
+
+```
+loadDataWithAddAndDelGraph(content: string, graph: string | NamedNode, additions: string, removals: string, format: string?): void
+```
+
+#### serializeDataWithAddAndDelGraph
+
+Returns the `graph` and its corresponding additions and removals graphs in the specified `format` (text/turtle by default).
+
+```
+serializeDataWithAddAndDelGraph(graph: NamedNode, format?: string): {
+    graph: string | undefined;
+    additions: string | undefined;
+    removals: string | undefined;
+}
+```
+
+#### serializeDataMergedGraph
+
+Returns the data that remains after applying additions and deletes to `graph` in the specified `format` (text/turtle by default).
+
+```
+serializeDataMergedGraph(graph: string | NamedNode, format?: string): string | undefined
+```
+
+#### parse
+
+Parses data from the specified `content` string (containing RDF) into this store. The data is structured in `format` mimeType (text/turtle by default) and is parsed into `graph`.
+
+```
+parse(content: string, graph: string | NamedNode, format?: string): void
+```
+
+#### match
+
+Returns the statements that match the pattern `subject`, `predicate`, `object` and `graph`.
+
+One or multiple of the terms of the triple can be set to `null` or `undefined`, turning them into wildcards. They will now match any value, e.g. `match(null, FOAF('knows'), null, null)` will match any person that knows someone. If we want to be more restrictive, we can say: `match(null, FOAF('knows'), PERSON('John'), null)` so that only the people that know John are returned. We can optionally specify the graph, e.g. `match(null, FOAF('knows'), PERSON('John'), profile)` so that only the people that know John according to my profile are returned.
+
+```
+match(subject: Quad_Subject | null, predicate: Quad_Predicate | null, object: Quad_Object | null, graph: string | NamedNode): Statement[]
+```
+
+#### any
+
+Returns the wildcard value of a triple matching the pattern `subject`, `predicate`, `object` and `graph`, or undefined if no match was found. If all arguments are specified and a match is found, true is returned.
+
+One term of the triple can be set to `null` or `undefined`, serving as a wildcard. They will now match any value, e.g. `match(null, FOAF('knows'), null, null)` will match any person that knows someone. If we want to be more restrictive, we can say: `match(null, FOAF('knows'), PERSON('John'), null)` so that only the people that know John are returned. We can optionally specify the graph, e.g. `match(null, FOAF('knows'), PERSON('John'), profile)` so that only the people that know John according to my profile are returned.
```

Comment:
Yes the forking-store implementation returns the first match. However, the [rdflib.js API](https://linkeddata.github.io/rdflib.js/doc/classes/Store.html#any) doesn't specify that it should be the first, but that it can be "a" match. I can specify that the implementation returns the first, but then the name `any` is a bit confusing. If nothing is found, indeed undefined is returned, maybe this is unclear in my explanation?

Does it return the full match (subject, predicate, object, graph) for you or only either the subject, predicate or object?

Yes the usage of `match` was an oversight.

Reply:
Logged the result of the `any()` method with the input of different parameters:
* fill in the predicate & graph => returns the subject of the node
* fill in the subject & graph => returns the first predicate of the node?
* fill in the object & graph => returns the subject of the node
* only fill in the graph => returns the subject of the node 

![image](https://github.com/redpencilio/forking-store/assets/36266973/cdcdfe30-0b38-4652-9611-e2420f68057e)

Am I testing this correctly or is it more confusing..

Reply:
I don't know the history of this method, it seems to be a bit of a funky one 😄. 

Looking at the implementation it only returns a single value, a Node corresponding to the first "wildcard". `true` if no "wildcard" was passed in, but a triple exists, or `undefined` if there were no matches.

So that seems to match with the description in the PR?